### PR TITLE
Remove design doc for PR review aid.

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: docs/MDSpaceDesign.md [hunk 1]
```diff
@@ -1,80 +0,0 @@
...
-1. Loosing the split factor (SplitInfo) as a parameter: automatically fixed as 2 for every axes of the box.
-2. Limited precision of the box hierarchy (the depth of tree): N/n for N bit Morton number and n dimensions, but there
-is the possibility to choose Morton number, that affects the performance.
-3. Fixed global box, the initial box can't grow with adding next portion of events, the boundaries should be defined
-once from instrument or sample geometry. 
-
-### Optimization notes
-
-The main bottleneck of the approach is memory access to MDEvent object. The best optimization for this is to make the
-MDEvent object as small as possible. To reach this the floating point coordinates can be thrown out the MDEvent structure
```
[the remark this answers is not in the document]
Comment (OwenArnold) [reply docs/MDSpaceDesign.md]: Can we also adopt a mdleanevent and drop the run and detector id information? Does that provide any worthwhile improvement?

File: docs/MDSpaceDesign.md [hunk 2]
```diff
@@ -1,80 +0,0 @@
...
-In fact there are only a few things to change in the current implementation of MD workspace:
-1. Add the index field in the box base class.
-2. Reimplement addEvent() and addEvents() functions. 
-
-### Drawbacks
-
-1. Loosing the split factor (SplitInfo) as a parameter: automatically fixed as 2 for every axes of the box.
-2. Limited precision of the box hierarchy (the depth of tree): N/n for N bit Morton number and n dimensions, but there
-is the possibility to choose Morton number, that affects the performance.
-3. Fixed global box, the initial box can't grow with adding next portion of events, the boundaries should be defined
```
Comment (OwenArnold) [inline docs/MDSpaceDesign.md]: Good point, but maybe reword slightly to say "Fixed global boundaries are necessary. Unlike the current MDEventWorkspace, the boxes will not dynamically grow when mdevents are added. The user will need to determine the static size before events are added." you could go on to say: "This is because morton numbers are determined by the extents in the world coordinates of the space".

File: docs/MDSpaceDesign.md [hunk 3]
```diff
@@ -1,80 +0,0 @@
...
-
-![MD workspace structure](MDSpace.png)
-
-
-*And adding new portion of events:*  
-
-![Merging thw new portion](merging_md_space.png) 
-
-### Changes to current implementation
-
```
[the remark this answers is not in the document]
Comment (OwenArnold) [reply docs/MDSpaceDesign.md]: I would go further and briefly mention the various options that have been considered in separate headings. The wider dev community will probably ask about this, Briefly explaining why those routes are not advantageous would be a good idea.

**Review by OwenArnold — CHANGES_REQUESTED** [review]
Great start. Thanks Igor.

File: docs/MDSpaceDesign.md [hunk 4]
```diff
@@ -1,80 +0,0 @@
...
-In fact there are only a few things to change in the current implementation of MD workspace:
-1. Add the index field in the box base class.
-2. Reimplement addEvent() and addEvents() functions. 
-
-### Drawbacks
-
-1. Loosing the split factor (SplitInfo) as a parameter: automatically fixed as 2 for every axes of the box.
```
[the remark this answers is not in the document]
Comment (DanNixon) [reply docs/MDSpaceDesign.md]: Yes, the factor just needs to be a power of 2 and constant for all axes.

(I hadn't considered the effects of setting this so left it hard coded at 2)

**Comment by igudich** [comment]
I would create anew one in the proper branch.
